### DISCUSSION TITLE
Can we make Danger JS not update build status for latest commit on PR of Bitbucket Cloud by default

### DISCUSSION THREAD

**Discussion by SumanDas94** [danger/danger-js#1297] [Q&A]
From what i have seen Danger JS on Bitbucket cloud automatically updates the build status for the latest commit of the PR after it is run.
I cannot find a way to toggle this off.
Is this always done by default or is there a mechanism to just let Danger comment on the PRs rather than setting a build status for it's latest commit as well ?

**Comment by orta** [comment]
I think this is the same question as https://github.com/danger/danger-js/issues/1293

**Reply by SumanDas94** [reply]
@orta yes `--no-publish-check` using this flag worked fine. Thanks
